config/secrets.yml

```diff
@@ -10,16 +10,91 @@
 # Make sure the secrets in this file are kept private
 # if you're sharing your code publicly.
 
+default: &default
```

Comment:
`common` implies sharing common code like what is happening here. I am fine with `default` too :smile: